docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -203,27 +207,27 @@ quarkus.oidc.auth-server-url=http://localhost:8180/realms/quarkus/
 quarkus.oidc.client-id=quarkus-app
 quarkus.oidc.credentials.jwt.key-file=privateKey.pem
 
-# This is a token key identifier 'kid' header - set it if your OpenID Connect provider requires it.
-# Note if the key is represented in a JSON Web Key (JWK) format with a `kid` property then
-# using 'quarkus.oidc.credentials.jwt.token-key-id' is not necessary.
+# This is a token key identifier 'kid' header - set it if your OpenID Connect provider requires it:
+# Note if the key is represented in a JSON Web Key (JWK) format with a `kid` property, then
+# using 'quarkus.oidc.credentials.jwt.token-key-id' is not necessary
 quarkus.oidc.credentials.jwt.token-key-id=mykey
 
-# Use RS512 signature algorithm instead of the default RS256
+# Use RS512 signature algorithm instead of the default RS256:
 quarkus.oidc.credentials.jwt.signature-algorithm=RS512
 
 # The token endpoint URL is the default audience value, use the base address URL instead:
 quarkus.oidc.credentials.jwt.audience=${quarkus.oidc-client.auth-server-url}
 
-# custom subject instead of the client id :
+# custom subject instead of the client ID:
```

Comment:
```suggestion
# custom subject instead of the client id
```
We have discussed it with Rolfe, I use capital `ID` only when talking about ID tokens.
Also, this is not a real doc but a comment in the properties, so here `:` looks unusual

Reply:
Lets keep `:` in some comments, sure, updating my suggestions to keep them as you intended

Reply:
Thanks for clarifying @sberyozkin. I reverted these instances of "ID" back to "id". I do think though that as we are talking about "client ID" in generic terms here and not the specific property name, that "ID" should be capitalized. However, for consistency with your other discussions, I will revert back to "id" here and touch base also with Rolfe.